src/main/scala/interplay/PlayBuildBase.scala

```diff
@@ -187,13 +193,19 @@ object PlayLibraryBase extends AutoPlugin {
   import PlayBuildBase.autoImport._
 
   override def projectSettings = Seq(
+    // In sbt 1 the scripted plugin is automatically enabled. It imports
```

Comment:
Shouldn't this be done only if `scalaBinaryVersion.value` is equal to `2.11`?

Reply:
May as well disable for both since the library is compiled for both Scala 2.11 and Scala 2.12.

It's a bit of a hack so we might change this anyway. I'm not confident it will work perfectly - but so far it seems OK. Akka didn't try for an automatic fix. Instead they call`disablePlugin(ScriptedPlugin)` manually in all their projects.

Reply:
I don't know if Akka build has plugins like we do (`PlayLibrary`, `PlaySbtLibrary`, etc), but if disabling `ScriptedPlugin` is better, maybe we should do it at these plugins (for example, like [we did here for `BintrayPlugin`](https://github.com/playframework/interplay/blob/master/src/main/scala/interplay/PlayBuildBase.scala#L57)).